languages/translations/de.json

```diff
@@ -84,12 +84,12 @@
         },
         "continue": {
             "default": {
-                "default": "{modifier} weiterfahren",
+                "default": "{modifier} abbiegen",
                 "name": "{modifier} weiterfahren auf {way_name}",
-                "destination": "{modifier} weiterfahren Richtung {destination}"
+                "destination": "{modifier} abbiegen Richtung {destination}"
             },
             "slight left": {
-                "default": "Leicht links weiter",
+                "default": "Scharf links",
```

Comment:
slight left should be "Leicht links" - Scharf means sharp, maybe a mixup in the translations